scripts/docker/buildpack-deps/Dockerfile.ubuntu1804

```diff
@@ -89,17 +89,16 @@ RUN set -ex; \
 	rm -f $TGZFILE;
 
 # HERA
-ARG HERA_HASH="622663cb3bc1fa07213c6e1fe8070c5c259096e75039a46d014b2a3448b5cf44"
-ARG HERA_MAJOR="0"
-ARG HERA_MINOR="3"
-ARG HERA_MICRO="0"
 RUN set -ex; \
-	HERA_VERSION="$HERA_MAJOR.$HERA_MINOR.$HERA_MICRO"; \
-	TGZFILE="hera-$HERA_VERSION-linux-x86_64.tar.gz"; \
-	wget https://github.com/ewasm/hera/releases/download/v$HERA_VERSION/$TGZFILE; \
```

Comment:
Why isn't the release build good enough? It has debugging off by default.

Reply:
Debugging is enable by default.

Reply:
Oh, indeed it is, that is a mistake upstream: https://github.com/ewasm/hera/blob/a772fafd7507fa0612837a8c3b1fdb6fd3f2674b/src/CMakeLists.txt#L31

It is not supposed to be on by default.

Reply:
Should I make a 0.3.2 release with that disabled, that may be better because then even the `install_dependencies` script will work as intended.

Reply:
Ah, great. Yes, I would say it would make sense to create a new release.